### DISCUSSION TITLE
Visual stimulus presentation time

### DISCUSSION THREAD

**Discussion by ThetaS** [bonsai-rx/bonsai#1389] [Q&A]
Hello.

This is my problem from an early unresolved post posted 2 weeks ago. During this time I've experimented with "FrameEventLogger" in a couple ways but none of these attempts satisfy the goal. The events logged are either only one trial or every frame update. In addition, no time info was included in the first scenario. 

I tried the solution from this post, but it does not work in my case. One possible concern is that I used a "repeat" node at the end of my rendering line.

https://github.com/orgs/bonsai-rx/discussions/1203

Hope this can be clarified.
[CSD.zip](https://github.com/bonsai-rx/bonsai/files/11589030/CSD.zip)

**Comment by bruno-f-cruz** [comment]
I think wrapping the stimuli logic inside its own observable might achieve what you want.
![image](https://github.com/bonsai-rx/bonsai/assets/7049351/7a1cf934-a2f3-4035-8577-8fb5c32db43b)

[CSD.zip](https://github.com/bonsai-rx/bonsai/files/11590264/CSD.zip)
